README.md

```diff
@@ -1,220 +1,40 @@
-[![Coverage][codecov-image]][codecov-url]
+# Important Links
+ - [CE  Team Confluence Documentation](https://openstax.atlassian.net/wiki/spaces/CE/overview)
+ - [CE Styles ZenHub Board](https://app.zenhub.com/workspaces/ce-styles-5cdc84baee941c07853fbbf2/board?repos=49744755,11369724,137909279,63083783)
+ - [Cookbook Repo](https://github.com/openstax/cookbook)
 
-**Note For Windows Users:** If your username contains whitespace, the setup and installation may not work. Create a new user without whitespace in the username in order to succesfully complete installation.
-
-# About 
-The cnx-recipes repo contains the recipes and styles framework. The recipe framework is used for content manipulation and the styles framework is used to style the content. The backend of the recipes framework is contained in the [cnx-easybake repo](https://github.com/openstax/cnx-easybake). 
+# About
+The ce-styles repo contains the  styles framework. The styles framework. If you're looking to manipulate the content during the baking stage, the recipes framework is contained in the [cookbook repo](https://github.com/openstax/cookbook).
```

Comment:
Got "the styles framework" doubled up here (or the description of the styles framework is missing)